.styleguidist/typography.md

```diff
@@ -1,4 +1,6 @@
-Suomifi-styleguide typography
+Suomifi-styleguide typography.
+
+Click the examples to copy the typography token name to clipboard.
```

Comment:
It seems to copy the tag and not token name:

`Heading 1 hero` --> `<Heading variant='h1hero' smallScreen='true'></Heading>`

Reply:
Fixed.